HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -184,7 +188,7 @@ extension InvoicesListViewModel: PaymentComponentsControllerProtocol {
     }
     
     func didReceivedErrorOnPaymentProviders(_ error: GiniHealthSDK.GiniHealthError) {
-        errors.append(error.localizedDescription)
+        errors.append(error.errorMessage)
         showErrorsIfAny()
```

Comment:
[Clarification]: last time @razvancapra that we don't need to show any errors in that case 🤔

Reply:
Yes, I will eliminate this method because we don't handle errors from fetching payment providers